05_stringer/alextmz/main_test.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_IPAddrString(t *testing.T) {
+	var tests = map[string]struct {
+		arg  IPAddr
+		want string
+	}{
+		"empty IPv4": {
+			IPAddr{},
+			"0.0.0.0"},
```

Comment:
Either put the closing brace on a line of its own, or use a more compact layout with everything on one line:
```
"empty IPv4": {IPAddr{}, "0,0.0.0},
...
```
The reason for putting the brace on its own line if you are making a vertical list, is that it makes every line consistently the same - they all have a comma at the end. If you extend the struct to add another field and add the extra element to each struct, you end up with a diff where you just have lines added. To modify your structure to add another element, you end up with a diff where the last line gets changed to change the `},` to `,` and then another line added.

Clearer diffs make reviewing easier, and makes it easier to spot mistakes, if for instance you accidentally delete a character on the line you would not otherwise touch.

Reply:
Makes sense. Done.